CONTRIBUTING.md

```diff
@@ -48,6 +48,8 @@ You can talk to our core team on [Discord](https://discord.gg/AJSX9DCSUA). And o
 * Moriz (@Mogge)
 * Wolle (@Tirokk)
 * Alex (@ogerly)
+* Hannes (@elweyn5803)
+* Mathias (@mahula)
```

Comment:
cool 👍🏼

Maeckes?

Reply:
Sure.
Added in 8c9fb9292a5d0edc4323ed1875980a33be10d775.